draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -362,12 +362,12 @@ The following table lists algorithms recommended by use-case:
 To find the suitable algorithm, lookup the requirement from above, with 
 the chosen curve in the below:
 
-| Curve  | Inj. Encoding | Random Oracle |
-|--------|---------------|------|
-| P-256 | Simple SWU {{simple-swu}} | FFSTV(SWU) 
-| P-384 | Icart {{icart}} | FFSTV(Icart) 
-| Curve25519 | Elligator2 {{elligator2}} | ... 
-| Curve448 | Elligator2 {{elligator2}} | ... 
+| Curve  | Prime | Characteristics | Inj. Encoding | Random Oracle |
```

Comment:
When rendered, this is a bit more information than I expected. I think we ought to drop these two new rows, as they don't seem to help when making an encoding decision.